pkg/devfile/parser/parse.go

```diff
@@ -217,6 +165,11 @@ func ParseDevfile(args ParserArgs) (d DevfileObj, err error) {
 		d.Ctx.SetToken(args.Token)
 	}
 
+	args.DevfileUtilsClient = NewDevfileUtilsClient()
+	if err != nil {
+		return d, fmt.Errorf("Error %v, URL cannot be parsed %s ", err, args.URL)
+	}
```

Comment:
I think can be removed?

Reply:
yes it looks like it was left there from another change that got removed